alodataset/coco_detection_dataset.py

```diff
@@ -16,7 +17,7 @@
 from torchvision.datasets.coco import CocoDetection
 
 from alodataset import BaseDataset
-from aloscene import BoundingBoxes2D, Frame, Labels
+from aloscene import BoundingBoxes2D, Frame, Labels, Panoptic
```

Comment:
Wonder if `Panoptic` is the right class name. Panoptic refer to the task it self (stuff + things) while this tensor actually refer to the data.  In imgaug, they refer to this type of class as : Segmentation Maps. Maybe `Segmentation` instead of `Panoptic` ? What do you think ?

Reply:
Ok, you are right. I will change the name